### DISCUSSION TITLE
Chandra  X-ray Observatory is made after which astrophysicist?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#877] [Galaxy Trivia] [answered]
1. Edwin Hubble
2. Subrahmanyan Chandrasekhar
3. Arthur Eddington  

& Date

**Comment by peterbaccaro** [comment, marked as answer]
The Chandra X-ray Observatory is named after the Indian-American astrophysicist **Subrahmanyan Chandrasekhar**. He made significant contributions to our understanding of stellar evolution and the physics of white dwarfs. Chandrasekhar's most famous work involved the study of the behaviour of matter under extreme conditions, particularly in relation to the behaviour of stars.

The Chandra X-ray Observatory was launched on **July 23, 1999**.
